orcid_hub/models.py

```diff
@@ -1772,6 +1776,9 @@ def load_from_json(cls, source, filename=None, org=None):
                             name = contributor.get("credit-name").get("value") if \
                                 contributor.get("credit-name") else None
 
+                            email = contributor.get("contributor-email").get("value") if \
```

Comment:
I reckon there are a few places, where you have to delve into the tree a few levels down. Perhaps it would pay off if it gets refactored our into ```get_val(el , *keys)``` that would pick up a value or return *None* if there is missing key on the way. Correct me if I'm wrong about it....